### PR TITLE
Remove Irma from messages

### PR DESCRIPTION
**Pull request by ssjunnebo**
Replaces mentions of Irma in email/log messages with "analysis cluster".

### CODE REVIEW

File: taca/analysis/analysis_nanopore.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def process_minion_qc_run(minion_run, sequencing_ongoing=False, nanoseq_ongoing=
 
                         else:
                             logger.warn('An error occurred during transfer of run {} '
-                                        'to Irma. Notifying operator.'.format(minion_run.run_id))
+                                        'to analysis cluster. Notifying operator.'.format(minion_run.run_id))
```
Comment (FranBonath) [inline taca/analysis/analysis_nanopore.py:134]: if you want me to be picky: change to 
to **the** analysis cluster

**Review by FranBonath — COMMENTED** [review]
Will you merge it or shall I? How is this usually handled?

File: taca/analysis/analysis_nanopore.py (hunk 1 above)
Comment (ssjunnebo) [reply taca/analysis/analysis_nanopore.py]: Nice catch 👍 Fixed it!

**Comment by ssjunnebo** [comment]
> Will you merge it or shall I? How is this usually handled?

Oh sorry, missed your comment! Yes please, merge if you're happy with the changes 🙂

**Pull request merged**